include/swift/Option/Options.td

```diff
@@ -935,11 +935,6 @@ def disable_batch_mode : Flag<["-"], "disable-batch-mode">,
 def wmo : Flag<["-"], "wmo">, Alias<whole_module_optimization>,
   Flags<[FrontendOption, NoInteractiveOption, HelpHidden]>;
 
-def force_single_frontend_invocation :
```

Comment:
I’m fine with removing the flag but let’s leave this option here as an alias for -whole-module-optimization. We have existing clients passing -force-single-frontend-invocation and we can’t break them.